cook-book/README.md

```diff
@@ -0,0 +1,3 @@
+# Cook Book
+
+* [Getting started](./getting-started)
```

Comment:
Strange path to dir, is it ok?

Also it's better to use absolute path

Reply:
What's strange?

Fixed to absolute.